src/main/resources/assets/additionalguns/models/item/awm.json

```
@@ -1,15 +1,16 @@
 {
-	"credit": "Made with Blockbench. Made by VIKTORWOOD",
+	"credit": "Made with Blockbench",
```

Please keep the names of the original authors :)
```suggestion
	"credit": "Made with Blockbench. Made by VIKTORWOOD",
```